README.md

```diff
@@ -0,0 +1,156 @@
+# To Do list:Version 1.4<!---Title of Project-->
+<a name="readme-top"></a>
+
+
+## React capstone project - Metrics webapp <!---Title of Project-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp<a name="about-project"></a> <!---Project title-->
+This React capstone project is about building a mobile web application to check a list of metrics that We've created making use of React and Redux. This website list the genres of popular shows and allows to see the shows descriptions.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+ React, Redux and CSS, and linter configuration setup.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo](https://odongo006.github.io/React-Metrics-Webapp/)
+
```

Comment:
- You have done a great job on this project. 👍🏽 
- I don't think your web app was deployed properly. Can you try to fix this, then add an updated live demo link on your readme?? 

This is what's being rendered 👇🏽 

|Your webapp homepage|
| :---:   | 
|<img width="1439" alt="Screenshot 2023-04-20 at 19 39 46" src="https://user-images.githubusercontent.com/99246359/233519473-6cb74198-5cf3-4854-a66c-a9eaf7b520db.png">  |